src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,22 @@
-export const UserInfo = () => {};
+type UsersItem = {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+};
+
+interface UserInfoProps {
+  user: UsersItem | undefined;
+}
+
+export const UserInfo: React.FC<UserInfoProps> = ({ user }) => {
+  return (
+    <>
+      {user && (
+        <a className="UserInfo" href={`mailto:${user.email}`}>
+          {user.name}
+        </a>
+      )}
+    </>
+  );
```

Comment:
```suggestion
interface UserInfoProps {
  user: UsersItem;
}

export const UserInfo: React.FC<UserInfoProps> = ({ user }) => {
  return (
        <a className="UserInfo" href={`mailto:${user.email}`}>
          {user.name}
        </a>
  );
```
let's remove the check here, and render only when user exists